exporter/awss3exporter/logbody_marshaler.go

```diff
@@ -0,0 +1,52 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package awss3exporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awss3exporter"
+
+import (
+	"bytes"
+	"fmt"
+
+	"go.opentelemetry.io/collector/pdata/plog"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+)
+
+type bodylogMarshaler struct{}
+
+func (*bodylogMarshaler) format() string {
+	return "log"
```

Comment:
should it be "log"?

Reply:
I modified the format to `.txt` for consistency with the document that states _'export the log body as a string.'_